src/devices/Ssd1351/README.md

```diff
@@ -0,0 +1,25 @@
+﻿# Solomon Systech Ssd1351
+
+## Summary
+The SSD1351 is a single-chip CMOS OLED driver with controller for organic/polymer light emitting diode dot-matrix graphic display system. It consists of 384 segments and 128 commons. This IC is designed for Common Cathode type OLED panel.
+
+## Device Family
+**SSD1351**: https://cdn-shop.adafruit.com/datasheets/SSD1351-Revision+1.3.pdf
+
+### Related Devices
+- [OLED Breakout Board - 16-bit Color 1.5"](https://www.adafruit.com/product/1431)
+- [OLED Breakout Board - 16-bit Color 1.27"](https://www.adafruit.com/product/1673)
+
+## Binding Notes
+
+This binding currently only supports commands and raw data.  Eventually, the plan is to create a graphics library that can send text and images to the device.
+
+### Connection Type
+
+The following connection types are supported by this binding.
+
+- [ ] I2C
```

Comment:
Does this mean that I2C is supported but not includded in the binding yet, or that I2C is not supported by the binding?

Reply:
I2C is not supported. I will remover the tickbox

Reply:
I have removed the I2C in the lastest PR